core/src/com/unciv/models/translations/TranslationFileWriter.kt

```diff
@@ -177,13 +176,24 @@ object TranslationFileWriter {
                         )
                 }
 
-                val lineToWrite = translationKey.replace("\n", "\\n") +
-                        " = " + translationValue.replace("\n", "\\n")
-                stringBuilder.appendLine(lineToWrite)
+                stringBuilder.appendTranslation(translationKey, translationValue)
             }
 
             countOfTranslatedLines[language] = translationsOfThisLanguage
 
+            // As a courtesy to translators, add in any deletions as comments
```

Comment:
I'm not sure this is a good idea. We ourselves add and remove lines a lot, so every time we deprecate a unique the old translation line will be added at the bottom? That's worse than just deleting it in my opinion

Reply:
True but a) we can fix spelling on the english side with complete impunity b) if it gets out of hand we'll introduce automatic cleanup. I _almost_ added a date...

Question is - should we use more imperative wording on the wiki or even other channels to get our translators to accept the cleanup responsibility?